src/addons/__tests__/ReactFragment-test.js

```diff
@@ -27,7 +27,7 @@ describe('ReactFragment', function() {
       x: <span />,
       y: <span />
     };
-    <div>{children}</div>;
+    void <div>{children}</div>;
```

Comment:
The alternative, IMHO is to disable no-unused-expression in `__tests__` only, by extending the grunt task to do the pattern matching itself, or by supporting multiple sets of eslintrc and eslintignore files.